github_token

```diff
@@ -0,0 +1 @@
+f6192b8d35518098fbb645ffd807a53749ffb00e
```

Comment:
Don't think this should be here, might be a good idea to regenerate your tokens too

Reply:
Yeah I deleted the access token, thanks.

Reply:
Was actually deleted before I accidentally pushed it here, but I'll reach out to GitHub support as well

Reply:
OK just verified the access token doesn't work by running:

```
curl -H "Authorization: token f6192b8d35518098fbb645ffd807a53749ffb00e" https://api.github.com
```

which returned

```
{
  "message": "Bad credentials",
  "documentation_url": "https://developer.github.com/v3"
}
```

So I must have deleted the token awhile back before it ever made it into this P.R

We're good 👍

Reply:
Yeah just make sure you delete the token in GH settings and should be good